src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -156,8 +156,14 @@ class TableView extends React.Component<TableViewProps, TableState> {
       return column.name;
     }
 
-    // TODO(leedongwei): Deprecate eventView and use state.columnSortBy
+    const sortKey = eventView.getSortKey(String(column.key), tableData.meta);
```

Comment:
I think we generally use template literals to cast to string instead of `String` constructor

Reply:
@billyvg eh really?  What's the rationale? 🤔

Reply:
It's just stylistic - here are the uses of `String`
![image](https://user-images.githubusercontent.com/79684/66865986-3a3b5d00-ef4d-11e9-969f-ac385476e08f.png)